index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title> Abel mobile setup </title>
+        <link rel="stylesheet" type="text/css" href="style.css">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+    </head>
+    <body>
+      <div class="home-page">
+        <header>
+          <nav id="nav">
+            <a class="logo" href="#">Abel</a> 
+
+            <nav class="hamburger"><img src="image/Union.png" alt="Menu"></nav>
+            <ul class="internal-link">
+               <li><a href="#my_work">Portfolio</a></li>
+               <li><a href="#about_me">About</a></li>
+               <li><a href="#form-section"> Contact </a></li>
+            </ul>
+
+          </nav>
+        </header>
+        <section id="home_presentation">
+           <h4 class="short-presentation">I’m Abel Fikre
+            Glad to see you!</h4>
+            <p>
+                I’m a Software Development Engineer and Cloud Architect! I can help you build a product ,
+                feature or website Look through some of my work and experience and publish it in the Cloud!
+                If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <footer>
+                <h3>LET'S CONNECT</h3>
+                <div id="contact">
+                  <ul>
+                    <li class="icon"><a href="https://twitter.com/abelfikre21" ><img src="image/twitter.png" alt="Twitter link"></a></li>
+                    <li class="icon"><a href="www.linkedin.com/in/ abel-fikre-03b173a" ><img src="image/Linkedin_icon.png" alt="github link"></a></li>
+                    <li class="icon"><a href="#" ><img src="image/M.png" alt="github link"></a></li>
+                    <li class="icon"><a href="https://github.com/abel12-pro" ><img src="image/github.png" alt="github link"></a></li>
+                    <li class="icon"><a href="#" ><img src="image/Vector.svg" alt="Angle link"></a></li>
+                  </ul>
+                </div>
+            </footer>
```

Comment:
- [ ] Please remove the `footer` tag because this is not a footer that is only used at last of the page so kindly remove this tag :+1: 
- [ ] As I mentioned above :point_up_2: please stick to the Figma design but your design is not the same as the Figma design so kindly fix it as well :+1: check below :point_down:  the comparison between your design and Figma design :slightly_smiling_face:

**Your Design**
![Screenshot from 2023-06-09 12-54-04](https://github.com/abel12-pro/Portfolio_mobileFirst/assets/45226848/0181682e-2a8f-4595-baf2-acf01c3f2e8c)

**Figma Design** 
![Screenshot from 2023-06-09 12-54-23](https://github.com/abel12-pro/Portfolio_mobileFirst/assets/45226848/30117d97-cdfc-460b-9f4d-07628821d113)